public/app/features/alerting/unified/components/contact-points/__snapshots__/useContactPoints.test.tsx.snap

```diff
@@ -218,7 +217,14 @@ exports[`useContactPoints when having oncall plugin installed and no alert manag
         },
       ],
       "name": "grafana-default-email",
-      "policies": undefined,
+      "policies": [
```

Comment:
The existing hook will always fetch the matching policies now - but that's okay, because the only time we _weren't_ asking for this was the simplified routing (which is whats been changed in this PR)